charmhelpers/contrib/openstack/utils.py

```diff
@@ -235,6 +238,8 @@
         ['2.24.0', '2.25.0']),
     ('victoria',
         ['2.25.0', '2.26.0']),
+    ('wallaby',
```

Comment:
The point of the new openstack-release package is that this should be unnecessary. on Hirsute+, the openstakc-package should be both available and installed on all openstack installs, and should tell is that this is running Wallaby

Reply:
But we still need this to work for focal, don't we (I mean right NOW).

Reply:
On focal-wallaby, the package should be present _in the cloud-archive_ for Wallaby.

Reply:
```
$ cmadison openstack-release
 openstack-release | 1.0ubuntu1~cloud0 | wallaby-proposed | focal-proposed | source
 ```

Reply:
I still wonder, for completeness, whether this should be added?

Reply:
For the dicts we no longer need to update (ie. the ones that openstack-release replaces) I think we should add a comment for wallaby+ to say that openstack-release package determines the openstack code name.